test/cases/testdata/time/test-time-0971.yaml

```
@@ -0,0 +1,46 @@
+cases:
+- data:
+  modules:
+  - |
+    package test
+
+    time_ns := 1670006453141828752
+
+    a = no_timezone {
+      no_timezone := time.timestamp(time_ns)
+    }
```

[nit] This could be condensed a bit, 
```suggestion
    a := time.timestamp(time_ns)
```

(also below, of course)